resources/views/service/registration.blade.php

```diff
@@ -75,22 +79,22 @@
             <div class="col">
                 <div>
                     <label for="cc-id">CC ID number:</label>
-                    <input type="text" id="cc-id">
+                    <input type="text" id="cc-id" name="cc_reference">
```

Comment:
Do you want to replace the lodash with a dash?

Reply:
name carries through to the validation and db field directly in this case, so i'd prefer to keep it.